draft-birkholz-scitt-software-supply-chain-use-cases.md

```diff
@@ -50,23 +53,41 @@ Generalized Software Supply Chain Use Case Descriptions
+Modern software applications are an intricate mix of first-party and third-party code, development practices and tools, deployment methods and infrastructure, and interfaces and protocols.
+The software supply chain comprises all elements associated with an application's design, development, deployment, and maintenance throughout its entire lifecycle.
+The complexity of software coupled with a lack of lifecycle visibility increases the risks associated with system attack surface and the number of cyber threats capable of harmful impacts, such as exfiltration of data, disruption of operations, and loss of reputation, intellectual property, and financial assets.
+There is a need for a platform architecture that will allow consumers to ascertain that suppliers maintained appropriate security practices without requiring access to proprietary intellectual property.
+SCITT-enabled products and analytics solutions will assist in managing compliance and assessing risk to help prevent and detect supply chain attacks across the entire software lifecycle while prioritizing data privacy.
+
+[^abs2-]
+
+[^abs2-]: Insert more detail on the development and deployment flows, threat landscape, and notional artifacts that allow security/compliance risk assessment.
+oducts and analytics solutions will assist in managing compliance and assessing risk to help prevent and detect supply chain attacks across the entire software lifecycle while prioritizing data privacy.
 
 ## Terminology {#terms}
 
 {::boilerplate bcp14-tagged}
 
 # Generic Problem Statement
 
-Supply chain security is paramount to protecting critical infrastructure, aerospace, and defense and avoiding impacts on security, the economy, public health, and safety. It has historically focused on risk management practices to safeguard logistics, meet compliance regulations, demand forecasts, and optimize inventory. While these elements are foundational to a healthy supply chain, an integrated cyber security-based perspective of the software supply chains remains broadly undefined. Recently, the global community has experienced numerous supply chain attacks by cybercriminals targeting weaknesses in software supply chains. As illustrated in {{lifecycle-threats}}, a software supply chain attack may leverage one or more lifecycle stages and directly or indirectly target the component.
```

Comment:
is this text intentionally removed?

Reply:
@yogeshbdeshpande, was this intentional, for brevity's sake?

Reply:
@SteveLasker @henkbirkholz I was just merging the same in the PR coming from the main line: Let me revisit the same!